src/yaml/kellydale2003/32691-cn-tower-new-years-day-version.yaml

```diff
@@ -0,0 +1,44 @@
+group: kellydale2003
+name: cn-tower-new-years-day-version
+version: "1.0"
+subfolder: 360-landmark
+info:
+  summary: CN Tower - New Year's Day version
+  description: |-
+    Happy New Year everyone!
+
+    Here is the CN Tower from Toronto, Ontario, Canada. This is the single tower. I will provide the full complex sometime in 2019. I've also noticed that I've made a couple of small errors and will update them accordingly. Follow this file and you'll receive a patch/update in the near future.
+
+    For now, here's the single tower version (the complex will come later). Festive to New Year's Day, this lot is CS$$ and has fireworks!
+
+    Now, you can add this overly massive telecommunications tower, turned tourist destination, to your city with festive style. This 553 meter (1,813+ feet) tower is the tallest building in Canada and was built to rival Toronto''s fast growing skyline.
+
+    The Toronto skyline began blocking telecommunications services and thus the CN Tower was required. When the tower was under construction, it looked like something out of a sci-fi movie!
+
+    Enjoy!
+
+    \- Kellydale2003
+
+    PS; Watch my thread for upcoming projects and releases!
+
+    [https://community.simtropolis.com/forums/topic/33505-kellydale2003s-bat-mine/?do=embed](https://community.simtropolis.com/forums/topic/33505-kellydale2003s-bat-mine/?do=embed)
+
+    [![CN-Tower-7-803x603-1517854112.jpg](https://www.simtropolis.com/objects/attachments/monthly_2019_01/CN-Tower-7-803x603-1517854112.jpg.3b36087fef1657cb175e2fcd196bd4b5.jpg)](https://www.simtropolis.com/objects/attachments/monthly_2019_01/CN-Tower-7-803x603-1517854112.jpg.3b36087fef1657cb175e2fcd196bd4b5.jpg)
+
+    [![CN-Tower-Stock.jpg](https://www.simtropolis.com/objects/attachments/monthly_2019_01/CN-Tower-Stock.jpg.b5ff257644d87f9c32b1be1d862f81dd.jpg)](https://www.simtropolis.com/objects/attachments/monthly_2019_01/CN-Tower-Stock.jpg.b5ff257644d87f9c32b1be1d862f81dd.jpg)
+  author: kellydale2003
+  website: https://community.simtropolis.com/files/file/32691-cn-tower-new-years-day-version/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_2019_01/cntower.jpg.eb7a6ee39c0fedff6274141c2b307374.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2019_01/cn3.jpg.1af9964efe666e06ffdc6ec16c78ef8b.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2019_01/cn4.jpg.bc5ae7c324b53431b9d9a0b7e28598d4.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2019_01/cn1.jpg.766e2d24efd34b29b352cfdc3ebe82ea.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2019_01/cn2.jpg.47be6d122e20ea0c884919de471b6ce2.jpg
+assets:
+  - assetId: kellydale2003-cn-tower-new-years-day-version
+
```

Comment:
Darknite only? Not mentioned in the description, but looks like it from the pictures.